generators/update-readme.js

```diff
@@ -0,0 +1,95 @@
+const path = require("path");
+const fs = require("fs");
+
+function baseReadme(
+  { componentName, propName, ruleName, referencePR, message },
+  inputExamples,
+  outputExamples
+) {
+  const readMePath = path.join(
+    require.resolve("@patternfly/pf-codemods").replace("index.js", ""),
+    "README.md"
+  );
+  const readMe = fs.readFileSync(readMePath, "utf8");
+  const splitReadMe = readMe.split("## Rules");
+  const splitRules = splitReadMe[1].split("\n### ");
+
+  const readMeAddition = `${ruleName} [(#${referencePR})](https://github.com/patternfly/patternfly-react/pull/${referencePR})
+
+${message}
+
+#### Examples
+
+In:
+
+\`\`\`jsx
+${inputExamples}
+\`\`\`
+
+Out:
+
+\`\`\`jsx
+${outputExamples}
+\`\`\`
+
+`;
+
+  splitRules.push(readMeAddition);
+  const newRules = splitRules.sort().join("\n### ");
```

Comment:
Right now this results in `datalist-remove-props` coming after `dataListCheck-warn-updated-callback`, and an import of `kebabToggle` coming after `KEY_CODES`. Would we want to ignore case when sorting, or do you think it makes more sense to take lettercase into account?

Not really a blocker, but I did notice it.

Reply:
Let me know if you want to try implementing it, otherwise I'm good merging in as-is and we can do followups for this.

Reply:
Wiki entry could work, or maybe a separate README somewhere

Reply:
Ah yeah, the contributing readme would make sense